openquakeplatform/openquakeplatform/isc_viewer/gs_data/featuretypes/isc_viewer_measure.xml

```diff
@@ -60,13 +71,35 @@
     <maxx>180.0</maxx>
     <miny>-90.0</miny>
     <maxy>90.0</maxy>
+    <crs>GEOGCS[&quot;WGS84(DD)&quot;, 
+  DATUM[&quot;WGS84&quot;, 
+    SPHEROID[&quot;WGS84&quot;, 6378137.0, 298.257223563]], 
+  PRIMEM[&quot;Greenwich&quot;, 0.0], 
+  UNIT[&quot;degree&quot;, 0.017453292519943295], 
+  AXIS[&quot;Geodetic longitude&quot;, EAST], 
+  AXIS[&quot;Geodetic latitude&quot;, NORTH]]</crs>
   </latLonBoundingBox>
   <projectionPolicy>FORCE_DECLARED</projectionPolicy>
   <enabled>true</enabled>
   <advertised>true</advertised>
+  <metadata>
+    <entry key="cachingEnabled">false</entry>
+    <entry key="JDBC_VIRTUAL_TABLE">
+      <virtualTable>
+        <name>isc_viewer_measure</name>
+        <sql>select the_geom, src_id, date, lat, lon, smajax, sminax, strike, epic_q, depth, depth_unc, depth_q, mw, mw_unc, mw_q, s, mo, fac, mo_auth, mpp, mpr, mrr, mrt, mtp, mtt, eventid from isc_viewer_measure order by date</sql>
+        <escapeSql>false</escapeSql>
+        <geometry>
+          <name>the_geom</name>
+          <type>Geometry</type>
+          <srid>4326</srid>
+        </geometry>
+      </virtualTable>
+    </entry>
+  </metadata>
   <store class="dataStore">
     <name>oqplatform</name>
-    <atom:link xmlns:atom="http://www.w3.org/2005/Atom" rel="alternate" href="http://127.0.0.1:8080/geoserver/rest/workspaces/oqplatform/datastores/oqplatform.xml" type="application/xml"/>
+    <atom:link xmlns:atom="http://www.w3.org/2005/Atom" rel="alternate" href="http://127.0.0.1:8000/geoserver/rest/workspaces/oqplatform/datastores/oqplatform.xml" type="application/xml"/>
```

Comment:
Since it's a GeoServer URL I would keep `8080` as port avoiding the use of the GeoNode proxy.